se2_planning/include/se2_planning/StateValidator.hpp

```diff
@@ -17,6 +17,10 @@ class StateValidator {
   virtual bool isStateValid(const State& state) const = 0;
   virtual void initialize();
   virtual bool isInitialized() const;
+  virtual bool isLocked() const;
+  virtual void lock();
+  virtual void unlock();
+  bool isLocked_ = false;
```

Comment:
shouldn't isLocked_ be private?

Reply:
Fixed in 51f0bace18ff73ef6c4ef0899163ed8b0206534f.